_config.yml

```diff
@@ -0,0 +1,35 @@
+title: AI Supported Continuous Integration
+author: AIOps
+# logo: logo.png
+execute:
+  execute_notebooks: false
+
+# Only build files specified by the _toc
+only_build_toc_files: false
```

Comment:
should this be `true`?

Reply:
It can be, but it was intentionally changed to "false" for this repo, since some of the files in this repo are not necessarily in TOC, but users may still stumble upon when exploring the book (links will be broken if they are not built). I don't mind changing it back, just LMK

Reply:
ok : ) leave it then.